chain-impl-mockchain/src/certificate/pool.rs

```diff
@@ -57,6 +57,8 @@ pub struct PoolPermissions(u64);
 pub type ManagementThreshold = u8;
 
 const MANAGEMENT_THRESHOLD_BITMASK: u64 = 0b11_1111; // only support 32, reserved one for later extension if needed
+
+#[allow(clippy::unusual_byte_groupings)]
 const ALL_USED_BITMASK: u64 =
     0b00000000_00000000_00000000_00000000_00000000_00000000_00000000_00111111;
```

Comment:
I wonder what's unusual about this. It may be better to just remove the redundant zeroes and leave the 8 LSBs.

Reply:
Grouping by 4 is the Clippy style.  I feel is ok that way, completely explicit about what is being used.